src/AMG.jl

```diff
@@ -1,8 +1,12 @@
 module AMG
```

Comment:
Could you also modify this module name to `AlgebraicMultigrid` ? This was AMG from the time it was ranjanan/AMG.jl